src/Symfony/Component/HttpKernel/EventListener/DumpListener.php

```diff
@@ -37,8 +39,16 @@ public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper
         $this->dumper = $dumper;
     }
 
-    public function configure()
+    public function configure(GetResponseEvent $event)
     {
+        // if the request is issued by curl, we do not enable HTML dumper;
+        if (false !== strpos($event->getRequest()->headers->get('user-agent'), 'curl/')) {
```

Comment:
Could we change this condition to enable this behavior only for requests not initiated via CLI?

```php
if ('cli' !== php_sapi_name()) {
    CliDumper::$defaultColors = true;
    CliDumper::$defaultOutput = 'php://output';

    return;
}
```